### PR TITLE
Form 8940: Schema updates#16045

### PR DESCRIPTION
**Pull request by zurbergram**
package.json
- update version to `3.127.0`

src/schemas/21-526EZ-allclaims/schema.js
- added `form8940` definition
- moved `hospitalizationHistory` into `form8940` definition

### CODE REVIEW

File: src/schemas/21-526EZ-allclaims/schema.js [hunk 1]
```diff
@@ -771,6 +748,234 @@ const schema = {
...
+              type: 'string'
+            },
+            occupationDuringMostEarnings: {
+              type: 'string'
+            },
+            previousEmployers: {
+              type: 'array',
+              items: {
+                type: 'object',
+                name: {
```
Comment (dmccurdybah) [inline src/schemas/21-526EZ-allclaims/schema.js]: Needs a nested `properties` field

File: src/schemas/21-526EZ-allclaims/schema.js [hunk 2]
```diff
@@ -771,6 +748,246 @@ const schema = {
...
+                    maxLength: 100
+                  },
+                  hospitalAddress: {
+                    type: 'object',
+                    properties: _.omit(
+                      ['addressLine3', 'country'],
+                      baseAddressDef.properties
+                    )
+                  }
+                }
```
Comment (imeecuison) [inline src/schemas/21-526EZ-allclaims/schema.js]: Do you mind moving hospitalizationDateRange after hospitalAddress so it shows up in the proper order on the page?

Comment (zurbergram) [reply src/schemas/21-526EZ-allclaims/schema.js]: `hospitalizationHistory` is going away switching to using `hospitalProvidedCare`

Comment (imeecuison) [reply src/schemas/21-526EZ-allclaims/schema.js]: okay. can you move the daterange so that it shows up as the last field on the page?

**Comment by zurbergram** [comment]
@department-of-veterans-affairs/code-reviewers-for-external-teams-group Could I get a code review on this at your earliest convenience. Thank you!

**Pull request merged**